.github/workflows/sonarcloud.yml

```diff
@@ -0,0 +1,75 @@
+###############################################################
+# Copyright (c) 2024 Contributors to the Eclipse Foundation
+#
+# See the NOTICE file(s) distributed with this work for additional
+# information regarding copyright ownership.
+#
+# This program and the accompanying materials are made available under the
+# terms of the Apache License, Version 2.0 which is available at
+# https://www.apache.org/licenses/LICENSE-2.0.
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+#
+# SPDX-License-Identifier: Apache-2.0
+###############################################################
+
+name: Sonarcloud
+on:
+  push:
```

Comment:
maybe you could still limit the triggering to src path like [here](https://github.com/eclipse-tractusx/ssi-credential-issuer/pull/8/files#diff-d7e1c74eda3e35f5cbf4cdfbefab1a0602329871a6f5e76f42d4335815a9a289R23)

Reply:
added the path filter to the workflow